app/storage.tsx

```diff
@@ -61,6 +71,70 @@ class Storage {
     }, null);
   }
 
+  // Insert expansion pack into the database
+  parseExpansionPack(expansionData) {
+    this.db.transaction((tx) => {
+      tx.executeSql(
+        'INSERT INTO expansionPackTable (pack_name, description, wtw, organisation) VALUES (?,?,?,?)',
+        [
+          expansionData.meta.pack_name,
+          expansionData.meta.description,
+          expansionData.meta.w3w,
+          expansionData.meta.organisation
+        ]
```

Comment:
we now have a version number too

Reply:
This updated will be sorted on another branch with a new example file.